package.json

```diff
@@ -11,22 +21,16 @@
     "copy:env": "node npm_tasks/copy-env.js",
     "test": "echo \"Test in webmaker-core\""
   },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/mozilla/webmaker-android.git"
-  },
-  "author": "",
-  "license": "MPL-2.0",
-  "bugs": {
-    "url": "https://github.com/mozilla/webmaker-android/issues"
-  },
   "dependencies": {
     "webmaker-core": "https://github.com/mozilla/webmaker-core.git#6c1b8afbe0c1c02af750c667f7bafcf6215c085a"
   },
-  "homepage": "https://github.com/mozilla/webmaker-android",
   "devDependencies": {
     "mkdirp": "^0.5.1",
     "ncp": "^2.0.0",
     "rimraf": "^2.4.1"
+  },
+  "engines": {
+    "node": "^4.1.0",
```

Comment:
This is here to prevent people from attempting to run NPM commands on webmaker-android, with an outdated version of NPM or Node outside of the Gradle build environment. If you trigger a Gradle build or use any commands with the Gradle prefix (I.e. gradle npmInstall), it will use the Node version Gradle downloaded.